src/utils/formUtils.js

```diff
@@ -96,8 +96,10 @@ export function eachComponent(components, fn, includeAll, path, parent) {
 
     if (!noRecurse) {
       if (hasColumns) {
-        component.columns.forEach((column) =>
-          eachComponent(column.components, fn, includeAll, subPath(), parent ? component : null));
+        component.columns.forEach((column) => {
```

Comment:
I'm not sure, but there's a chance we may even want to do this with the other calls to `eachComponent` in this function. I didn't include that here, as I'm just trying to be conservative.